fem/estimators.hpp

```diff
@@ -304,6 +304,83 @@ class L2ZienkiewiczZhuEstimator : public ErrorEstimator
 
 #endif // MFEM_USE_MPI
 
+/** @brief The LpErrorEstimator class compares the solution to a known
+    coefficient.
```

Comment:
I think it's a little confusing that this class is not really an estimator, because it computes the "exact" error (assuming the integration is correct). But I'm not strongly against, it's just something to consider.

The documentation could perhaps include the intended usage (as in the PR description): "This can be used, for example, to adapt a mesh to a non-trivial initial condition in a time dependent problem."

Reply:
Good point!